docs/reference/content/reference/ecmascriptnext/connecting.md

```diff
@@ -21,7 +21,7 @@ const assert = require('assert');
   const url = 'mongodb://localhost:27017/myproject';
   // Database Name
   const dbName = 'myproject';
-  const client = new MongoClient(url);
+  const client = new MongoClient(url, {useNewUrlParser: true});
```

Comment:
nit: `{useNewUrlParser: true}` needs spaces between the curly braces so it should look like `{ useNewUrlParser: true }`